includes/polling/ports.inc.php

```diff
@@ -764,7 +764,7 @@
         }//end if
 
         // We don't care about statistics for skipped selective polling ports
-        if (!empty($port['skipped'])) {
+        if (!empty($port['skipped']) || ($port['ifOperStatus'] == "down" && $port['ifOperStatus_prev'] == "down" && $this_port['ifOperStatus'] == "down" && $this_port['ifLastChange'] == $port['ifLastChange'])) {
```

Comment:
Bonjour ! 
Looks good, even if I wonder if some of these tests are in fact redundants. But I need to have a deeper look. That beeing said, you could probably comment this a little more. I would also suggest to go with "elseif" statement : makes it a little bit more verbose, but that would separate the selective polling code from this ifOperStatus=down optimisation. 
Merci

Reply:
What is the gains from this change?  (Polling time change)

Reply:
It depends on the disk activity in term of IO per second. 
One of our server was running out of IO. **Polling total time changed from 2200s to 1500s with 320 polled equipments** (~320 IOPS write disk perf)
No significant performance improvment observed on another server with the same number of equipments but disk activity was not overchaged (~600 IOPS write disk perf)